src/sentry/integrations/vsts/integration.py

```diff
@@ -66,12 +67,18 @@ def __init__(self, projects, *args, **kwargs):
         )
 
 
-class VSTSIntegrationProvider(IntegrationProvider):
+class VstsIntegration(Integration):
+    def get_client(self, access_token):
```

Comment:
is the `access_token` something you can get from the integration model (which will be `self.model` here)? my only concern is that you're changing the signature without changing the base class, which will likely cause issues when you actually try to use it